README.md

```diff
@@ -24,3 +17,21 @@ npm run dev
 ```
 
 This app should run on `http://localhost:3000/`
+
+### Vercel
```

Comment:
I would rethink these sections.

1. First time set up
2. Running app locally
3. Running tests
4. Deploying app to Vercel (optional)
5. Setting up SonarSource (optional)

I say optional for the last two since they are more related to your project than your future customers. 

SonarSource would be 100% related to any future contributors.